web/template/home.gohtml

```diff
@@ -290,8 +292,266 @@
+                                                            <i class="fas fa-location-pin"></i>
+                                                            <span x-text="livestream.LectureHall.Name"></span>
+                                                        </a>
+                                                    </template>
+                                                    <span class="tum-live-badge bg-red-500 text-white uppercase">
+                                                    <i class="fa-solid fa-satellite-dish"></i>
+                                                    Live
+                                                </span>
+                                                </div>
+                                                <a :href="`/w/${livestream.Course.Slug}/${livestream.Stream.ID}`">
+                                                    <div :style="`background-image:url('/api/stream/${livestream.Stream.ID}/thumbs/live')`"
+                                                         class="h-full tum-live-thumbnail"></div>
+                                                </a>
+                                            </div>
+                                            <div class="px-3">
+                                                <a class="title text-xl"
+                                                   :href="`/w/${livestream.Course.Slug}/${livestream.Stream.ID}`"
+                                                   x-text="livestream.Stream.Name"></a>
+                                                <span class="description text-sm mt-2"
+                                                      x-text="livestream.Stream.Description"></span>
+                                                <span class="date text-sm"
+                                                      x-text="livestream.Stream.UntilString()"></span>
+                                            </div>
+                                        </article>
+                                    </template>
+                                </section>
+                            </template>
+                        </div>
+                        <template x-if="courseStreams.hasElements()">
+                            <section class="tum-live-course-view-item">
+                                <header>
+                                    <h3 class="font-bold">VODs</h3>
+                                    <section class="space-x-2">
+                                        <button type="button" @click="sortNewestFirst()"
+                                                class="tum-live-button tum-live-button-tertiary"
+                                                :class="{'active' : isNewestFirst() }">
+                                            Newest first
+                                        </button>
+                                        <button type="button" @click="sortOldestFirst()"
```

Comment:
For me, this doesn't change the order, it is always newest first for me. Also seen in the test database for course "Einführung in Brauereiwesen"

Reply:
Should be fixed now. Thanks for pointing out! 👍🏻

Reply:
Works now. 👍